src/main/scala/outwatch/dom/Compat.scala

```diff
@@ -27,4 +27,43 @@ trait Handlers {
   def createHandler[T](defaultValues: T*): IO[Pipe[T, T]] = Handler.create[T](defaultValues: _*)
 }
 
-object Handlers extends Handlers
\ No newline at end of file
+object Handlers extends Handlers
+
+trait AttributesCompat {
+  lazy val `class` = className
```

Comment:
Which attributes should be deprecated? I kind of like the idea to mainly rely on scala-dom-types defined names, but I guess we can make some exceptions: Do we want to keep `for` and `class` or should we deprecate them? I think it should be either both or none. What do you think?

Reply:
decided to keep `for` and `class`. the rest is deprecated.